frontend/packages/console-app/src/components/quick-starts/data/explore-pipeline-quickstart.ts

```diff
@@ -8,94 +8,46 @@ export const explorePipelinesQuickStart = {
-        },
-        recapitulation: {
-          success: "You've just installed the Pipelines Operator!",
-          failed: 'Check your work to make sure that ',
-        },
-      },
-      {
-        title: `Explore your pipeline run`,
-        description: `### To install the Pipelines Operator:
-1. In the Administrator perspective, go to the OperatorHub from the Operators section of the navigation.
-2. Use the filter at the top of the page to search for the OpenShift® Pipelines Operator.
-3. If the card has an Installed label on it, the Operator has already been installed.  You won't be able to perform the following steps, and should cancel out of the Quick Start
-4. Click on the card to open the Operator details.
-5. At the top of the side panel, click Install.
-6. Fill out the Operator Subscription form, choose the channel matching your OpenShift cluster, and click Install. You can leave all of the prefilled options if you’d like.
-7. After a few minutes, the OpenShift® Pipelines Operator should appear in your Installed Operators list.
-8. Click it to see its details.`,
+1. From the **Administrator** perspective in the console navigation panel, click **Operators > OperatorHub**.
+2. In the **Filter by keyword** field, type \`OpenShift Pipelines Operator\`.
+3. If the tile has an Installed label, the Operator is already installed. Proceed to the next quick start to create a Pipeline.
+4. Click the **tile** to open the Operator details.
+5. At the top of the OpenShift Pipelines Operator panel that opens, click **Install**.
+6. Fill out the Operator subscription form by selecting the channel that matches your OpenShift cluster, and then click **Install**.
+7. On the Installed Operators page, wait for the OpenShift Serverless Operator's status to change from **Installing** to **Succeeded**. `,
         review: {
-          instructions: `#### To verify that the Pipelines Operator was successfully installed:
-1. Go to the Installed Operators page from the Operators section of the navigation.
-2. Check if OpenShift Pipelines Operator appears in the list.
-3. Check the status of the Operator.\n  4. Momentarily, the status should become Succeeded.
+          instructions: `#### To verify that the OpenShift Pipelines Operator is installed:
+1. From the **Operators** section of the navigation, go to the **Installed Operators** page.
+2. Verify that the **OpenShift Pipelines Operator** appears in the list of Operators.
 
-Is the status Succeeded?`,
-          taskHelp: 'Try walking through the steps again to properly install Pipelines Operator.',
+In the status column, is the status of the OpenShift Serverless Operator **Succeeded**?`,
+          taskHelp: `This task isn’t verified yet. You might have missed a step! Try the task again, or [read more](https://docs.openshift.com/container-platform/4.6/pipelines/installing-pipelines.html#op-installing-pipelines-operator-in-web-console_installing-pipelines) about this topic.`,
```

Comment:
I believe we agreed to remove "You might have missed a step!"

FYI @bmignano

Reply:
Sorry I could have missed that, remove that from all of them now @maxwelldb